src/access/AccessControl.h

```diff
@@ -341,13 +341,20 @@ class AccessControl
         // Iteration
         virtual CHIP_ERROR Entries(EntryIterator & iterator, const FabricIndex * fabricIndex) const { return CHIP_NO_ERROR; }
 
+        // Check
+        // Return CHIP_NO_ERROR if allowed, CHIP_ERROR_ACCESS_DENIED if denied,
+        // CHIP_ERROR_NOT_IMPLEMENTED to use the default check algorithm (against entries),
+        // or any other CHIP_ERROR if another error occurred.
+        virtual CHIP_ERROR Check(const SubjectDescriptor & subjectDescriptor, const RequestPath & requestPath,
+                                 Privilege requestPrivilege)
+        {
+            return CHIP_ERROR_ACCESS_DENIED;
```

Comment:
Shouldn't this default to NOT_IMPLEMENTED?  Then the delegates that want to use entries just override the other bits, and the ones that don't override this method, as opposed to forcing all delegates that want to allow anything to override `Check`.

Reply:
That works too.

Reply:
It means it'll be more work to determine "denied" in the unconfigured case, but those are generally test cases anyways (because a real system should be configured with *something* whatever that something is).

Reply:
It also means, by default, a PASE session will get the admin access. If we want true "everything is denied if we didn't set anything up" then it should explicitly return "denied" otherwise it's OK to proceed with checking entries.

Reply:
Per Slack discussion, we should leave this as-is, so implementors of access control delegates don't end up granting PASE access without explicitly opting into it (by either checking for it themselves or explicitly returning NOT_IMPLEMENTED to get the default behavior).